libs/stocks/data-access-price-query/src/lib/+state/price-query.effects.ts

```diff
@@ -24,9 +24,9 @@ export class PriceQueryEffects {
       run: (action: FetchPriceQuery, state: PriceQueryPartialState) => {
         return this.httpClient
           .get(
-            `${this.env.apiURL}/beta/stock/${action.symbol}/chart/${
+            `${this.env.apiURL}${priceQueryConstants.apiUrlConst}${action.symbol}${priceQueryConstants.apiUrlChartConst}${
```

Comment:
replaced strings with constants which can be updated anytime in future without affecting the effects file.